os/android/drmhwctwo.cpp

```diff
@@ -264,15 +264,20 @@ HWC2::Error DrmHwcTwo::HwcDisplay::GetChangedCompositionTypes(
   return HWC2::Error::None;
 }
 
-HWC2::Error DrmHwcTwo::HwcDisplay::GetClientTargetSupport(uint32_t /*width*/,
-                                                          uint32_t /*height*/,
-                                                          int32_t /*format*/,
+HWC2::Error DrmHwcTwo::HwcDisplay::GetClientTargetSupport(uint32_t width,
+                                                          uint32_t height,
+                                                          int32_t format,
                                                           int32_t dataspace) {
-  if (dataspace != HAL_DATASPACE_UNKNOWN &&
-      dataspace != HAL_DATASPACE_STANDARD_UNSPECIFIED)
+  if (width == display_->Width() && height == display_->Height() &&
+      format == HAL_PIXEL_FORMAT_RGBA_8888 &&
+      dataspace == HAL_DATASPACE_UNKNOWN) {
+      return HWC2::Error::None;
+    }
+  else if (!(display_->CheckPlaneFormat(format)) &&
```

Comment:
The check here should be ||, if format is unsupported or dataspace is not either of them. Also, this can be separate if since instead of else.
You can run git diff | clang-format-diff-3.5 -p 1 -style=file before committing, so code gets indented properly.